src/patternfly/components/List/examples/List.md

```diff
@@ -77,6 +77,77 @@ cssPrefix: pf-c-list
 {{/list}}
 ```
 
+### With horizontal rules
+```hbs
+{{#> list list--type="ul" list--modifier="pf-m-plain pf-m-bordered"}}
+  <li>Donec blandit a lorem id convallis.</li>
+  <li>Integer in volutpat libero.</li>
+  <li>Donec a diam tellus.</li>
+  <li>Aenean nec tortor orci.</li>
+  <li>Vivamus maximus ultricies pulvinar.</li>
+{{/list}}
+```
+
+### With small icons
+```hbs
+{{#> list list--type="ul" list--modifier="pf-m-plain"}}
+  {{#> list-item}}
+    {{#> list-item-icon}}
+      <i class="fas fa-book-open" aria-hidden="true"></i>
```

Comment:
in instances like this where we want to ensure all of the icons have the same width, font awesome has a utility class `.fa-fw` that will give an icon a fixed width. You can apply that to all of the icons that you want to have a fixed width relative to each other.
```suggestion
      <i class="fas fa-book-open fa-fw" aria-hidden="true"></i>
```